views/js/runner/plugins/content/modalFeedback/modalFeedback.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016  (original work) Open Assessment Technologies SA;
+ *
+ * @author Alexander Zagovorichev <zagovorichev@1pt.com>
+ */
+
+/**
+ * @see http://www.imsglobal.org/question/qtiv2p1/imsqti_implv2p1.html#section10008 modalFeedback
+ */
+
+define([
+    'jquery',
+    'lodash',
+    'module',
+    'taoTests/runner/plugin',
+    'taoQtiTest/runner/plugins/content/dialog/itemInlineMessage',
+    'taoQtiTest/runner/plugins/content/dialog/itemAlertMessage',
+    'ui/autoscroll'
+], function ($, _, module, pluginFactory, inlineMessage, alertMessage, autoscroll) {
+    'use strict';
+
+    /**
+     * Modal or inline type of the messages
+     */
+    var inlineMode;
+
+    /**
+     * Form of the feedback
+     * by default dialog (modal) form
+     */
+    var messagePlugin;
+
+    /**
+     * All feedback messages
+     */
+    var renderedFeedbacks;
+
+    /**
+     * modalFeedback was resolved and all components were destroyed
+     */
+    var isDestroyed;
+
+    /**
+     * Method which should be halted after modalFeedbacks confirmation action
+     */
+    var nextStep;
+
+    function destroyFeedback(feedback) {
+
+        var removed = false;
+        _.remove(renderedFeedbacks, function (storedFeedback) {
+
+            var found = storedFeedback === feedback;
+            if (found) {
+                removed = true;
+            }
+            return found;
+        });
+
+        if (removed) {
+            feedback.destroy();
+
+            if (!renderedFeedbacks.length) {
+                nextStep();
+            }
+        }
+    }
+
+    function defineMode(inline) {
```

Comment:
This is for qunit tests https://github.com/oat-sa/extension-tao-testqti/blob/0a303b3f4cd1b5fbb32e48bc29c5b8cebc55be43/views/js/test/runner/plugins/content/modalFeedback/test.js#L437-L437 
I can determine what exactly I want show, even if in the module config will be default value